### PR TITLE
Specific cd user

### PR DESCRIPTION
**Pull request by tjaeschke**

### CODE REVIEW

**Comment by tjaeschke** [comment]
First part for core

**Comment by tjaeschke** [comment]
relates to https://github.com/opendevstack/ods-project-quickstarters/issues/289

**Comment by tjaeschke** [comment]
Would you please check the implementation?

File: create-projects/create-cd-jenkins.sh [hunk 1]
```diff
@@ -69,10 +69,12 @@ tailor_update_in_dir "${SCRIPT_DIR}/ocp-config/cd-jenkins" \
     "--namespace=${PROJECT_ID}-cd" \
     "--param=PROXY_TRIGGER_SECRET_B64=${PIPELINE_TRIGGER_SECRET}" \
     "--param=PROJECT=${PROJECT_ID}" \
+    "--param=CD_USER_ID_B64=${CD_USER_ID_B64}" \
```
Comment (michaelsauter) [inline create-projects/create-cd-jenkins.sh]: Have you checked that this overrides the param as defined in `ods-core.env`? I guess it will work ...

**Pull request merged**